test/nand.js

```diff
@@ -0,0 +1,37 @@
+import eq from './shared/eq';
+import { nand } from '../src';
+
+describe('nand', function() {
+  it('should work on booleans', function() {
+    eq(nand(true, true), false);
+    eq(nand(false, true), true);
+    eq(nand(true, false), true);
+    eq(nand(false, false), true);
+  });
+
+  it('should work on numbers', function() {
+    eq(nand(1.0, 1.0), false);
+    eq(nand(0.0, 1.0), true);
+    eq(nand(1.0, 0.0), true);
+    eq(nand(0.0, 0.0), true);
+  });
+
+  it('should work on a combination of things', function() {
+    eq(nand(true, 1.0), false);
+    eq(nand(null, true), true);
+    eq(nand(1.0, undefined), true);
+    eq(nand(0.0, false), true);
+    eq(nand(null, null), true);
+    eq(nand(null, undefined), true);
+  });
+
+  it('should support currying', function() {
+    const trueNand = nand(true);
```

Comment:
Pls import RA in following way `import * as RA from '../src';`

can we simplify into ?
```js
it('should support currying', function() {
  eq(RA.nand(true, true), false);
  eq(RA.nand(true)(true), false);   
});